executor/write_test.go

```diff
@@ -642,6 +642,12 @@ func (s *testSuite) TestLoadData(c *C) {
 	r := tk.MustQuery(selectSQL)
 	r.Check(nil)
 
+	sc := ctx.GetSessionVars().StmtCtx
+	originIgnoreTruncate := sc.IgnoreTruncate
+	defer func() {
```

Comment:
Because this test uses the same `ctx` or `StmtCtx` is OK, other tests I can't guarantee.